migrations/sqlite/2018-12-25-164502_media-cover-deletion/down.sql

```diff
@@ -0,0 +1,4 @@
+-- This file should undo anything in `up.sql`
+
+ALTER TABLE posts DROP CONSTRAINT posts_cover_id_fkey;
+ALTER TABLE posts ADD CONSTRAINT posts_cover_id_fkey FOREIGN KEY (cover_id) REFERENCES medias(id);
```

Comment:
unfortunately, you cannot drop or add constraints on sqlite tables without doing the create temporary table dance

Reply:
Travis made me notice it. I knew for columns, I through it worked at least for constraints (it's only drop, add work just fine if I remember well)